doc_files/iter8_install.md

```diff
@@ -68,38 +62,24 @@ iter8-analytics              ClusterIP   172.21.106.44   <none>        80/TCP
+To enable users to see Prometheus metrics that pertain to their canary releases or A/B tests, iter8 provides a Grafana dashboard template. To take advantage of Grafana, you will need to import this template. To do so, first make sure you can access Grafana. In a typical Istio installation, you can port-forward Grafana from Kubernetes to your localhost's port 3000 with the command below:
 
 ```bash
 kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=grafana -o jsonpath='{.items[0].metadata.name}') 3000:3000
 ```
 
 After running that command, you can access Grafana's UI at `http://localhost:3000`.
 
-To import iter8's dashboard template for Istio, execute the following two commands:
-
-```bash
-export DASHBOARD_DEFN=https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/config/grafana/istio.json
-
-curl -s https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/hack/grafana_install_dashboard.sh \
-| /bin/bash -
-```
-
-If you are using iter8 with Knative, use these two commands instead:
-
-```bash
-export DASHBOARD_DEFN=https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/config/grafana/knative.json
-
-curl -s https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/hack/grafana_install_dashboard.sh \
-| /bin/bash -
-```
+Depending on the version of Istio telemetry (`v1` or `v2`) and Kubernetes (prior to 1.16 and 1.16+) you are using, you will need to import a different Grafana dashboard. Follow [these instructions](grafana.md) to import the appropriate dashboard template.
 
 ## Uninstall _iter8_
 
-If you want to uninstall all _iter8_ components from your Kubernetes cluster, first delete all instances of `Experiment` from all namespaces. Then you can delete iter8 by running the following command:
+If you want to uninstall all _iter8_ components from your Kubernetes cluster, first delete all instances of `Experiment` from all namespaces. Then, you can delete iter8 by running the following command, adjusting the URL based on the release you are uninstalling.
 
 ```bash
-kubectl delete -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/install/iter8-controller.yaml
+kubectl delete -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/<release>/install/iter8-controller.yaml
 ```
 
+In the URL above, replace the string `<release>` with the string for your desired release. For example, a valid release string is `v0.0.1`.
+
 Note that this command will delete the `Experiment` CRD and wipe out the `iter8` namespace, but it will not remove the iter8 Grafana dashboard if created.
```

Comment:
Should we also comment that it does not delete any instances of the Experiment resource (which will be hard to remove). Perhaps, add "Note that this command will delete the `Experiment` CRD but not any instances of `Experiment` resources. It is recommended that they be deleted before deleting `iter8`.

Reply:
Take a look at the first paragraph of the section `Uninstall iter8`. It asks the users to first remove the CRs from the cluster. See if you are fine with that text or have further suggestions.

Reply:
good enough